### DISCUSSION TITLE
[1.3.0] OnRenderImage 안돼요

### DISCUSSION THREAD

**Discussion by Deytooh** [naverz/zepeto-studio-kor#822] [4. General]
안녕하세요.

OnRenderImage는 모바일에서 작동이 안돼고있어요 ㅎㅎ
Unity 에서 잘 돼고 빌드 하면...알죠.

` OnRenderImage(source: RenderTexture, destination: RenderTexture) {
        console.log("on render that!!");
        Graphics.Blit(source, destination, this.material);
    }`
    
    Graphics.Blit(source,destination) 괜찮은데 this.material 추가하면 ... 안돼요 !

**Comment by Zepeto-TechEvangelist** [comment]
Hello. 
If English is more convenient for you, you can leave a question in English on our global community!
https://github.com/naverz/zepeto-studio-global/discussions
